src/07_slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1:2])
 
 # Output the second-to-last element: 9
-print()
+print(a[4:5])
 
 # Output the last three elements in the array: [7, 9, 6]
-print()
+print(a[3:6])
```

Comment:
you can also start at a negative index!  printing a[-3: ] would have the same effect and would be a little more robust if the list length changes.  <3